src/google/protobuf/compiler/command_line_interface.cc

```diff
@@ -188,8 +188,25 @@ bool TryCreateParentDirectory(const string& prefix, const string& filename) {
 // Get the absolute path of this protoc binary.
 bool GetProtocAbsolutePath(string* path) {
 #ifdef _WIN32
-  char buffer[MAX_PATH];
-  int len = GetModuleFileName(NULL, buffer, MAX_PATH);
+  wchar_t wide_buffer[MAX_PATH];
+  // Note: the returned length does not include terminating null character.
+  int len = GetModuleFileNameW(NULL, wide_buffer, MAX_PATH);
```

Comment:
It doesn't make much sense. The returned value of GetModuleFileNameA is always MBCS (Multi-Byte Character Set) and not necessarily UTF-8 hence there is no reason to use GetModuleFileNameA regardless of UNICODE macro definition.
See [MSDN](https://msdn.microsoft.com/en-us/library/cwe8bzh0.aspx) for details.